lib/metadata/metadataUtils.js

```diff
@@ -132,35 +140,36 @@ function metadataGetObject(bucketName, objectKey, versionId, log, cb) {
  * @return {undefined} - and call callback with params err, bucket md
  */
 function metadataValidateBucketAndObj(params, log, callback) {
-    const { authInfo, bucketName, objectKey, versionId, requestType, preciseRequestType, request } = params;
-    const canonicalID = authInfo.getCanonicalID();
+    const { authInfo, bucketName, objectKey, versionId, requestType, request } = params;
     async.waterfall([
         function getBucketAndObjectMD(next) {
```

Comment:
could also take the opportunity to update the `function ...` syntax to `(...) =>`
```suggestion
        (next) => {
```